operators/src/processing/expression/codegen.rs

```diff
@@ -325,13 +333,12 @@ impl ToTokens for Assignment {
 #[derive(Debug, Clone)]
 pub enum Parameter {
```

Comment:
does an enum with one variant make sense?

Reply:
Yes, it is extensible.

Reply:
but, yagni

Reply:
Yagni means I wouldn't create it. But it was already there, so we (1) needed it and (2) was less work to do it this way.